bin/ci-expected-results

```diff
@@ -1,17 +1,54 @@
 # final verification
 # we start from an empty index
-total=$(curl -v localhost/search | jq ".total")
-if [ ! "$total" ]; then
-    total=-1
-fi
 
-echo "Total results in the index: $total"
-expected_results_api_dummy=41
-if [ ! "$total" -eq "$expected_results_api_dummy" ]; then
-    echo "There should be exactly $expected_results_api_dummy results in the index"
-    if [ "$total" -gt "$expected_results_api_dummy" ]; then
+actual_total=$(curl -v localhost/search | jq ".total")
+if [ ! "$actual_total" ]; then
+    actual_total=-1
+fi
+echo "Total results in the index: $actual_total"
+expected_total=44
+if [ ! "$actual_total" -eq "$expected_total" ]; then
+    echo "There should be exactly $expected_total results in the index"
+    if [ "$actual_total" -gt "$expected_total" ]; then
         echo "(have more test fixtures been added to api-dummy?)"
     fi
     exit 2
 fi
+
+echo "CHECK: Search for article id"
+actual_first_id=$(curl -v "localhost/search?for=04395&page=1&per-page=10&sort=relevance&order=desc&use-date=default" | jq ".items[0].id")
+expected_first_id='"04395"'
+if [[ $actual_first_id != $expected_first_id ]]; then
```

Comment:
I would prefer these:

`if [[ $actual_first_id != $expected_first_id ]]; then` 

to look like this:

`if [ "$actual_first_id" != "$expected_first_id" ]; then`

double brackets are for extended types of comparisons. They're a little bit cleverer with slightly different parsing rules and in the cases used in this changeset they're unnecessary.

Reply:
PR here: https://github.com/elifesciences/search/pull/306

if you approve then the changes will be merged into this PR